lib/rubycritic/generators/html/templates/overview.html.erb

```diff
@@ -20,12 +20,16 @@
         <!--Churn vs Complexity graph-->
         <div class="col-xs-7 col-sm-7 col-md-6 col-lg-8">
           <div class="Graph_Cards fadeIn">
+            <% if Config.source_control_present? %>
+              <div id="churn-vs-complexity-graph-container" class="chart-container"></div>
+            <% else %>
+              <div id="churn-error">Please use a source-control system to calculate Churn.</div>
```

Comment:
That's a very misleading error message. I don't think it should be an error at all since the user did nothing wrong.

How about changing this to

>> We can't show you churn because this project does not seem under a source code management system like git or perforce at the moment. This doesn't mean that anything is wrong, it just means certain features like this one are not avaiable.